src/selectors/getConsoleHistory.js

```diff
@@ -1,3 +1,11 @@
-export default function getConsoleHistory(state) {
-  return state.get('console');
+export function getConsoleHistory(state) {
```

Comment:
This breaks the pattern of `export default` in existing selectors but it seemed overly split up to have each of these functions in their own file. In my own limited experience I've been defining these inline